### PR TITLE
Create installation.md

### PR DESCRIPTION
**Pull request by kratkiewiczk**

### CODE REVIEW

**Review by Schaubt — APPROVED** [review]
looks good 2 me

File: installation.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Installation Instructions      
+     
+No installation necessary, if you wish to quickly use our app we have deployed it on firebase here: wsu-retraceme.web.app       
+
+If you wish to run the project on local hardware, first download the project directory on the master branch, then save the directory wherever you like, and finally enter the public folder and launch the app from Uindex.html
```
Comment (w1-ll) [inline installation.md:5]: I would recommend you to write it in stel # format

Comment (w1-ll) [inline installation.md]: Write the step is # format

**Review by w1-ll — CHANGES_REQUESTED** [review]

**Review by w1-ll — APPROVED** [review]
Much better

**Pull request merged**